### PR TITLE
Update title of Basic List with custom role and class section of BasicList demo (#5648)

### LINKED ISSUE #5648: Update the title of the `Basic List with custom role and class` section of the `BasicList` demo page
The current title of this section is too long and not appropriate:

image

This title should be changed to `Customization`.

The demo file name is `BitBasicListDemo.razor` in the `Bit.BlazorUI.Demo.Client.Core` project.

### PR DESCRIPTION
**Pull request by Neha5679**
this closes #5648

### CODE REVIEW

File: checks.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"enabled":true,"categories":{}}
```
Comment (msynk) [inline checks.json:1]: pleaser remove this redundant file.

**Review by msynk — CHANGES_REQUESTED** [review]

**Comment by Neha5679** [comment]
I changed from my side

**Review by msynk — CHANGES_REQUESTED** [review]
please delete the `checks.json` file from your branch and then update the PR.

**Comment by Neha5679** [comment]
updated

**Review by msynk — APPROVED** [review]

**Pull request merged**